src/store.js

```diff
@@ -371,6 +371,11 @@ class AsyncStorageStore {
    */
   async handleMigrationOldRegisteredTokens() {
     const oldTokens = this.getItem('wallet:tokens');
+    if (!oldTokens) {
```

Comment:
Is there any place in the app that we expect this key (`REGISTERED_TOKENS_KEY `) to exist in the storage?

Instead of doing this, why don't we just do this?

```
const oldTokens = this.getItem('wallet:tokens') ?? [];
```

So the code will continue working and will save an empty object in the storage in case there are no custom tokens.

Reply:
Nowhere in the code we expect this key to exist: all `getItem` calls have a protection in place.

That said, I agree with your suggestion: it's more dev friendly to always have the key in storage. Refactored on [e3668d4](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/380/commits/e3668d4737c70033854aa9d134bd37ec7dbc22e6)